src/js/files/components/UploadOverlay.js

```diff
@@ -36,15 +38,54 @@ const UploadOverlayList = styled(BoxGroupSection)`
     padding: 0;
 `;
 
+const UploadInfo = styled.div`
+    padding-bottom: 5px;
+`;
+
+const FormattedRemainingUploadTime = styled.div`
```

Comment:
Remember convention:
```suggestion
const StyledUploadInformation = styled.div`
```